### DISCUSSION TITLE
Devices disconnecting several times per day. This is how I solved it.

### DISCUSSION THREAD

**Discussion by Chippi** [Koenkk/zigbee2mqtt#12467] [Show and tell]
## My setup
Platform: Raspberry Pi 4, Raspbian 32-bit
Coordinator: Conbee II
Z2M installation: Docker

## TLDR
Switch to ZigBee channel 25

## Long version
I've been running Z2M for a year without any problems, but a month ago problems started out of nowhere. Suddenly my devices kept disconnecting multiple times a day, so I began troubleshooting. I litteraly tried everything: 

- Switching to an older Z2M version, 
- Testing different extension cables and angles on my coordinator
- Switching my WIFI network channels & turning off WIFI
- Disconnecting all devices that could cause interference
- Raspberry Pi software updates
- Coordinator firmware updates & resets
- A bunch of more stuff, can't remember (todo)

I live in an apartment and the area is pretty crowded so I downloaded an wifi analyzer (Ubiquiti WiFiman) to see if there was any congestion. So was it? Yeah, the 2.4Ghz band is overloaded with maybe 20 other networks and I knew there's some new neighbours that recently moved in and one of them has a fucking ion cannon access point that just screws over the whole 2.4 Ghz band.

I began looking for a solution to the congestion problem and I stumbled across this article. https://www.metageek.com/training/resources/zigbee-wifi-coexistence/

Apparently channel 25/26 has less congestion because those channels lives at the edge of the last non-overlapping 2.4Ghz channel. So I tried switching my whole Z2M network to ZigBee channel 25 and voila, everything is ultra stable and responsive. Not a single disconnection for a month.

I hope this can solve issues for other people in the same situation.

![image](https://user-images.githubusercontent.com/6388933/168334143-4589692c-f8e7-4fb1-8c1e-8b2d79c263ad.png)
